src/NClient/NClient.Common/Helpers/Converters.cs

```diff
@@ -0,0 +1,21 @@
+using System;
+
+namespace NClient.Common.Helpers
+{
+    public static class Converters
```

Comment:
This class must be internal

Reply:
Disagree. Can you explain? I think we can provide this solution of converts for user, aren't I right?

Reply:
`NClient.Common` is not a package, so this converter is available only via the `NClient.Providers.Serialization.Protobuf.ProtobufNet` package. Why do users of this package need a converter?

Reply:
Ok